yt/frontends/ramses/data_structures.py

```diff
@@ -723,7 +731,9 @@ def read_rhs(f, cast):
             self.omega_lambda = rheader["omega_l"]
             self.omega_matter = rheader["omega_m"]
             self.hubble_constant = rheader["H0"] / 100.0  # This is H100
-        self.max_level = rheader["levelmax"] - self.min_level - 1
+        self.max_level = (
+            min(self._force_max_level, rheader["levelmax"]) - self.min_level - 1
```

Comment:
If I'm reading this right, `self._force_max_level` is meant to use the save number convention as RAMSES, not yt ?
I'm inclined to think that yt's convention should be followed uniformly on the user side and conversions (+/-1) should only happen under the surface as implementation details. Whichever convention you decide to settle for, it would be good to explicitly document it, at _least_ as an inline comment.

Reply:
You're right. The way it is implemented here is that `self._force_max_level` uses RAMSES convention, which means that level=1 is the base grid (1x1x1) and you can go up to `levelmax`.
The yt convention starts numbering at the finest level where the grid is "full".

As a RAMSES user I'd rather use RAMSES convention because that's what everybody would expect, but I would be happy with other suggestions as well.

Maybe one last solution would be to provide the ability to feed `_force_max_level` as a tuple, like `(force_max_level, "yt")` `(force_max_level, "ramses")`. Either we make the tuple notation explicit, or we add a warning when using the non-tuple format saying whichever convention was picked. Wdyt?

Reply:
That would work. I've been trying to think of a better way but I couldn't.